pkg/boot/measurement.go

```diff
@@ -69,6 +69,9 @@ func (mr *MeasuringReader) ReadRecord() (cpio.Record, error) {
 		switch rec.Name {
 		case "signature":
 			_, err = mr.signature.ReadFrom(uio.Reader(rec))
+			if err != nil {
```

Comment:
if _, err := glab; err != nil {
...
}
?

Reply:
Ack, I'll fix this

Reply:
No, this is currently deliberate. Please do not add this. I'll add a comment later.

Reply:
Sounds good. I'll remove this file and we can re-review it later.

Reply:
I'll make sure to add this linter to CI.